src/components/FilterPlaylist.tsx

```diff
@@ -76,6 +77,7 @@ function FilterPlaylist() {
             onClose={() => setModalOpen(false)}
             onNo={() => {console.log("no"); setModalOpen(false)}} />
         }
+        <div className="mb-4"></div>
```

Comment:
Could we get the same visual without having to use an empty div somehow?

Reply:
Great catch failed to recognized that the bottom margin could be used in the parent div holding the playlist cards. The floating div is removed!